### PR TITLE
PulsePoint Lite adapter

### PR DESCRIPTION
**Pull request by anand-venkatraman**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [X] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change
Lite version of PulsePoint adapter. The current PulsePoint adapter downloads a JS library, and then executes the bid requests. The Lite adapter does not depend on JS library. This will replace the existing PulsePoint adapter, after comparing the results, in a subsequent release.

Additionally,  a minor change to the endpoint of the JS lib for the old Adapter, is also included.

- test parameters for validating bids
```
{
bidder: 'pulsepointLite',
   params: { 
      cf: '300X250',
      cp: 521732,
      ct: 76835
   }
}
```
- avenkatraman@pulsepoint.com
- [X] official adapter submission - PulsePoint

### CODE REVIEW

File: src/adapters/pulsepointLite.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      cwu: pg.pg,
+      cwr: pg.ref,
+      dw: document.documentElement.clientWidth,
+      cxy: document.documentElement.clientWidth + ',' + document.documentElement.clientHeight,
+      tz: new Date().getTimezoneOffset(),
+      ln: (navigator.language || navigator.browserLanguage || navigator.userLanguage || navigator.systemLanguage)
+    };
+  }
+
+  function pageUrl() {
```
Comment (jaiminpanchal27) [inline src/adapters/pulsepointLite.js]: I would recommend using these function utils.getTopWindowUrl() and utils.getTopWindowLocation()

File: src/adapters/pulsepointLite.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      return {
+        pg: location.href,
+        ref: document.referrer
+      };
+    }
+  }
+
+  function bidResponseAvailable(bidRequest, rawResponse) {
+    if (rawResponse) {
+      var bidResponse = JSON.parse(rawResponse);
```
Comment (jaiminpanchal27) [inline src/adapters/pulsepointLite.js]: wrap this in try catch

File: src/adapters/pulsepointLite.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import {createBid} from 'src/bidfactory';
+import {addBidResponse} from 'src/bidmanager';
+import {logError} from 'src/utils';
+import {ajax} from 'src/ajax';
+import {STATUS} from 'src/constants';
+
+function PulsePointLiteAdapter() {
+
+  var bidUrl = window.location.protocol + '//bid.contextweb.com/header/tag?';
```
Comment (jaiminpanchal27) [inline src/adapters/pulsepointLite.js]: Using const/let is recommended.

Comment (anand-venkatraman) [inline src/adapters/pulsepointLite.js]: updated.

File: src/adapters/pulsepointLite.js (hunk 1 above)
Comment (anand-venkatraman) [inline src/adapters/pulsepointLite.js]: fixed

File: src/adapters/pulsepointLite.js (hunk 2 above)
Comment (anand-venkatraman) [inline src/adapters/pulsepointLite.js]: wrapped in ```try/catch```

**Comment by anand-venkatraman** [comment]
@jaiminpanchal27 Comments addressed.

**Comment by jaiminpanchal27** [comment]
LGTM. Thanks @anand-venkatraman

**Comment by jaiminpanchal27** [comment]
This is merged into master. Please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html). Thank you for contributing

**Pull request merged**